duckduckgo/Android
# Feature/phishing url fix

Task/Issue URL: https://app.asana.com/0/inbox/608920331025313/886417873773490/886557353517165
Tech Design URL: 
CC: 

**Description**:
The `5.12.1` hotfix restored basic functionality to Lollipop users, but removed protection from a niche phishing attack vector. This PR restores that protection again.

**Steps to test this PR**:
### Test 1
1. From a new tab, ensure the overflow menu options are suitably disabled
1. Perform a search, or open a webpage, ensure the menu options are available for use as expected (share, add bookmark etc...)

### Test 2
1. Visit https://geekboy.ninja/p0c/spoof-bar.html
1. Ensure the URL does NOT display `gmail` when it is loading

⚠️ Test both tests above on a Lollipop device, and a newer one.


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
